lib/shopify-cli/log.rb

```diff
@@ -21,10 +21,23 @@ def tail(n)
       buf = ""
       while buf.count("\n") <= n
         buf = read(TAIL_BUF_LENGTH) + buf
-        seek(2 * -TAIL_BUF_LENGTH, SEEK_CUR)
+        sk = 2 * -TAIL_BUF_LENGTH
+        if sk < -size
+          seek(0)
+          buf = read(size - buf.size) + buf
+          break
+        else
+          seek(sk, SEEK_CUR)
+        end
       end
+      ret = buf.split("\n")[-n..-1]
+      [ret, ret.join("\n").bytesize + 1]
+    rescue Errno::EINVAL
+      clear
```

Comment:
should you return `[[], 0]` here? Other wise it will break when you do 
```
        new_events, pos = events.tail(200)
        new_events.select!
```
later

Reply:
Actually, I fixed it so this error doesn't happen anymore (caused by sending a negative index longer than the buffer to `seek`), so this code can just be removed.